pkgs/applications/editors/android-studio/default.nix

```diff
@@ -0,0 +1,49 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
Please take each parameter you need "{ coreutils, findutils, ..." instead of "pkgs ? import <nixpkgs> {}".